### PR TITLE
Add information about installing gx into readme

### PR DESCRIPTION
**Pull request by Kubuxu**
License: MIT
Signed-off-by: Jakub Sztandera <kubuxu@gmail.com>

### CODE REVIEW

**Comment by whyrusleeping** [comment]
@RichardLitt wanna take a look at this one (its readme related)

**Comment by whyrusleeping** [comment]
very strange failure: https://circleci.com/gh/ipfs/go-ipfs/2988

Its not related to the code here, but i'm really not sure what caused it.

File: README.md [hunk 1]
```diff
@@ -73,6 +73,11 @@ export PATH=$PATH:$GOPATH/bin
 
 (If you run into trouble, see the [Go install instructions](https://golang.org/doc/install)).
 
+#### Prerequisite 2: Install gx and gx-go
+
+You can try using your package manager for it but it is unlikely it is there.
+Other way is to run `make toolkit_upgrade` before running `make install` in the next step.
```
Comment (RichardLitt) [inline README.md]: You can try using another package manager, but gx and gx-go are unlikely to be available elsewhere currently. Another way to install is to run `make toolkit_upgrade` before running `make install` in the next step.

Comment (whyrusleeping) [inline README.md]: maybe go with "currently, the recommended way to install them is to run `make toolkit_upgrade`..."

**Comment by Kubuxu** [comment]
Updated

**Comment by whyrusleeping** [comment]
@Kubuxu thanks, LGTM

**Pull request merged**